docs/sdk/client-side-sdks/ios.md

```diff
@@ -9,6 +9,13 @@ The iOS Client SDK for DevCycle! This SDK uses our Client SDK APIs to perform al
 and bucketing for the SDK, providing fast response times using our globally distributed edge workers
 all around the world.
 
+The SDK is available as a package on CocoaPods, Carthag and Swift Package Manager. It is also open source and can be viewed on Github.
+
+[![CocoaPods compatible](https://img.shields.io/cocoapods/v/DevCycle.svg)](https://cocoapods.org/pods/DevCycle)
```

Comment:
is this link supposed to just redirect to https://devcycle.com/?

Reply:
yeah... Cocoa Pods just redirect back to our site